tests/Assembly/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -20,6 +32,18 @@ contract Assembly {
       }
     }
   }
+
+  function caseAssembly() {
+    assembly {
+      switch value
```

Comment:
uhm, shall we have 2 spaces to indent the "case" inside the switch?

Reply:
You mean that the cases should be more indented than the switch? Yeah, I agree with that, and that's what prettier does in javascript.

Reply:
Done.

Reply:
awesome! thanks so much